src/test/java/com/rockset/client/api/TestCustomDocumentsApi.java

```diff
@@ -40,8 +40,11 @@ public void testConversionToRocksetObject() {
 
     // java.sql
     document.put("sql_timestamp", new Timestamp(1656417600000L));
-    document.put("sql_date", new Date(1656417600000L));
-    document.put("sql_time", new Time(1656417600000L));
+    document.put("sql_date", new Date(2022-1900, 6-1, 28));
+    document.put("sql_time", new Time(5, 0, 0));
```

Comment:
I've updated these two lines to use the explicit, but deprecated, constructors for two reasons:
* the millisecond constructors converts the time to local JVM time zone, so tests failed: `1656417600000L` is actually 2022-06-28T12:00:00 UTC, but the assertions expected it to be 05:00:00! The test would fail when not run on PST :/
* The explicit constructor makes expected values easier to reason about. Mapping `1656417600000L` to `2022-06-28` isn't obvious.

The downside is that these constructors are deprecated, and have non obvious semantics (e.g. year is `<actual year> - 1900`).

Reply:
> The test would fail when not run on PST

oh wow. Can you add a comment about this with link to java docs: https://docs.oracle.com/javase/7/docs/api/java/sql/Date.html#Date(long); thank you!